src/test/kotlin/it/pagopa/aca/ObjectTestUtils.kt

```diff
@@ -1,17 +1,16 @@
 package it.pagopa.aca
 
 import it.pagopa.aca.domain.Iupd
+import it.pagopa.generated.aca.model.NewDebtPositionRequestDto
 import it.pagopa.generated.aca.model.ProblemJsonDto
 import it.pagopa.generated.apiconfig.model.IbanEnhancedDto
 import it.pagopa.generated.apiconfig.model.IbanLabelDto
 import it.pagopa.generated.apiconfig.model.IbansEnhancedDto
-import it.pagopa.generated.gpd.model.PaymentOptionModelResponseDto
-import it.pagopa.generated.gpd.model.PaymentPositionModelBaseResponseDto
-import it.pagopa.generated.gpd.model.PaymentPositionModelDto
+import it.pagopa.generated.gpd.model.*
 import java.time.OffsetDateTime
 import org.springframework.http.HttpStatus
 
-object AcaTestUtils {
+object ObjectTestUtils {
```

Comment:
Why this refactoring? Object is a too generic name

Reply:
We will have too many classes with very similar names

Reply:
fixed 👉🏻 80f5953d8a7d22e5d6dcfbdf5193831132169a42